### PR TITLE
New term: Morton Neuroma

### LINKED ISSUE #5541: Morton neuroma
**Preferred term label: Morton neuroma**

**Synonyms: Morton metatarsalgia, interdigital neuritis, Morton entrapment, interdigital neuralgia, interdigital neuroma, interdigital nerve compression syndrome, and intermetatarsal neuroma**

**Definition (PMID: 29262171): compressive neuropathy of the forefoot interdigital nerve chiefly due to compression and irritation at the plantar aspect of the transverse intermetatarsal ligament. It is not a true neuroma as the condition is degenerative rather than neoplastic. The most common location for Morton neuroma is between the 2nd and 3rd metatarsals**

**Parent term: nervous system disorder --> peripheral nervous system disorder --> neuroma**

**Children terms N/A**

**Your nano-attribution (ORCID): 0000-0001-7151-1615**

### PR DESCRIPTION
**Pull request by sabrinatoro**
Closes #5541

### CODE REVIEW

File: src/ontology/components/mondo-tags.owl [hunk 1]
```diff
@@ -38684,5 +38682,5 @@
 
 
 
-<!-- Generated by the OWL API (version 4.5.6) https://github.com/owlcs/owlapi -->
+<!-- Generated by the OWL API (version 4.5.25.2023-02-15T19:15:49Z) https://github.com/owlcs/owlapi -->
```
Comment (nicolevasilevsky) [inline src/ontology/components/mondo-tags.owl:38685]: @sabrinatoro I'm not sure why these changes were added. Do you know if this is okay?

Comment (sabrinatoro) [reply src/ontology/components/mondo-tags.owl:38685]: I think that it is related to the fact that I removed a "exclude from QC" annotation, and it automatically updated the "tags" component. So I "think" it is ok. But maybe @matentzn can confirm?

Comment (matentzn) [reply src/ontology/components/mondo-tags.owl:38685]: Absolutely this is the correct behaviour!

Comment (nicolevasilevsky) [reply src/ontology/components/mondo-tags.owl:38685]: good to know, thank you!

**Review by nicolevasilevsky — APPROVED** [review]
